cli/js/40_testing.js

```diff
@@ -151,26 +150,18 @@ const OP_DETAILS = {
   "op_ws_send_pong": ["send a message on a WebSocket", "closing a `WebSocket` or `WebSocketStream`"],
 };
 
-function collectReliableOpMetrics() {
-  let metrics = core.metrics();
-  if (metrics.opsDispatched > metrics.opsCompleted) {
-    // If there are still async ops pending, we drain the event loop to the
-    // point where all ops that can return `Poll::Ready` have done so, to ensure
-    // that any ops are ready because of user cleanup code are completed.
-    const hasPendingWorkerOps = metrics.ops.op_host_recv_message && (
-      metrics.ops.op_host_recv_message.opsDispatched >
-        metrics.ops.op_host_recv_message.opsCompleted ||
-      metrics.ops.op_host_recv_ctrl.opsDispatched >
-        metrics.ops.op_host_recv_ctrl.opsCompleted
-    );
-    return opSanitizerDelay(hasPendingWorkerOps).then(() => {
-      metrics = core.metrics();
-      const traces = new Map(core.opCallTraces);
-      return { metrics, traces };
-    });
+let opIdHostRecvMessage = Number.MAX_SAFE_INTEGER;
```

Comment:
As we're using #[smi] on the op below, I'd suggest using an explicit `-1` here as the sentinel. These will be transparently wrapped and the error may not be evident.